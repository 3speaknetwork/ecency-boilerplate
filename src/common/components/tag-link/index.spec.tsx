import React from 'react';

import TagLink from './index';
import TestRenderer from 'react-test-renderer';
import {createBrowserHistory} from 'history';

it('(1) Default render', () => {
    const props = {
        history: createBrowserHistory(),
        location: {
            pathname: '/'
        },
        global: {
            tag: 'bitcoin',
            filter: 'hot'
        },
        communities: {list: {}},
        tag: 'bitcoin',
        children: <span>bitcoin</span>,
        fetchCommunity: () => {
        }
    };

    // @ts-ignore
    const renderer = TestRenderer.create(<TagLink {...props}/>);
    expect(renderer.toJSON()).toMatchSnapshot();
});
