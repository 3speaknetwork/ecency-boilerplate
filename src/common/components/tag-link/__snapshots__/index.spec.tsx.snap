// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(1) Default render 1`] = `
<a
  href="/hot/bitcoin"
  onClick={[Function]}
>
  bitcoin
</a>
`;
